### DISCUSSION TITLE
Where to actually get ahold of sales for GraalVM?

### DISCUSSION THREAD

**Discussion by cyraid** [oracle/graalpython#251] [General]
We don't have an Oracle Subscription because we wanted to evaluate GraalVM Python, but I wanted to speak to someone on the phone but there's no phone # or anything, and I tried calling multiple numbers but the Oracle Sales and others haven't even heard of GraalVM.

It's unfortunate that Python is in an experimental state, I was hoping to use it but we'll stick with Jep for now. Perhaps we can evaluate the product in the future sometime! :)

**Comment by shaunsmith** [comment]
I'd be happy to connect you with the Oracle sales teams dedicated to GraalVM if you want to email me? shaun dot smith at oracle dot com.  For community support you can join our Slack workspace: https://www.graalvm.org/slack-invitation/
